documentation/todo-app/src/components/todoMain.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
```

Comment:
Your React components should have a filename which is (exactly) the same as the component name. We always capitalise React component names, so it's also conventional to capitalise the filename. In this case, you should have:

    Header.js => not headers.js
    Logo.js => not logo.js
    TodoItem.js => not todoItems.js (it is a singular item, not a list)
    TodoMain.js => not todoMain.js